pkg/obfuscator/ip.go

```diff
@@ -36,26 +44,51 @@ func (o *ipObfuscator) replace(s string) string {
 	ipv4matches := o.ipv4pattern.FindAllString(output, -1)
 	for _, m := range ipv4matches {
 		if ip := net.ParseIP(m); ip != nil {
-			output = strings.ReplaceAll(output, m, obfuscatedStaticIPv4)
-			o.ReportReplacement(m, obfuscatedStaticIPv4)
+			var replacement string
+			switch o.replacementType {
+			case schema.ObfuscateReplacementTypeStatic:
+				replacement = obfuscatedStaticIPv4
+			case schema.ObfuscateReplacementTypeConsistent:
+				if replacement = o.HasReplacement(m); replacement == "" {
+					o.ipv4count++
+					replacement = fmt.Sprintf(consistentIPv4Template, o.ipv4count)
```

Comment:
we don't need to solve this in this PR, but we need to make it look like a proper IP address - ideally with some random octets instead of a counter.

Reply:
Why? What's the benefit of doing that?

Reply:
actually you're right, I thought it was in the spec like that.